tests/api-suite-info/05-get-latest-state-1.t

```diff
@@ -63,24 +67,18 @@ TEST_NAME="${TEST_NAME_BASE}-validate"
 run_ok "${TEST_NAME}" cylc validate "${SUITE_NAME}"
 
 cylc run --hold "${SUITE_NAME}"
-SRV_D="$(cylc get-global-config --print-run-dir)/${SUITE_NAME}/.service"
-HOST="$(sed -n 's/^CYLC_SUITE_HOST=//p' "${SRV_D}/contact")"
-PORT="$(sed -n 's/^CYLC_SUITE_PORT=//p' "${SRV_D}/contact")"
-AGENT="cylc/${CYLC_VERSION:-$(cylc --version)} prog_name/cylc-test-battery uuid/$(uuidgen)"
+UUID="$(uuidgen)"
 
-# Call 1, full, no "mean_main_loop_interval" as suite started on hold
+# Call 1, full
 run_ok "${TEST_NAME_BASE}-1" \
-    env no_proxy=* curl -A "${AGENT}" -v --cacert "${SRV_D}/ssl.cert" \
-    --digest -u "cylc:$(<"${SRV_D}/passphrase")" \
-    "https://${HOST}:${PORT}/get_latest_state"
+    cylc client -n --set-uuid="${UUID}" 'get_latest_state' "${SUITE_NAME}"
```

Comment:
Could (eventually) create a function for a ``run_ok`` test of incremental name calling this exact command (as is done throughout this test file) after ``sleep 1`` or other activity to make this test a bit more intuitive & tidy .